lte/gateway/c/core/oai/test/mme_app_task/test_mme_app_config_partial_tai_list.cpp

```diff
@@ -0,0 +1,532 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+
+extern "C" {
+#include "mme_config.h"
+#include "dynamic_memory_check.h"
+#include "log.h"
+}
+
+uint8_t ncon_tac[] = {1,  2,  4,  6,  7,  8,  10, 11, 12, 14, 15,
```

Comment:
you can define this as protected/private field of MMEConfigTest class and set its value in SetUp() method.